src/webots/gui/WbMainWindow.cpp

```diff
@@ -2295,7 +2295,8 @@ void WbMainWindow::updateOverlayMenu() {
         action->setCheckable(true);
         action->setChecked(device->isOverlayEnabled());
         action->setEnabled(!device->isWindowActive());
-        action->setProperty("renderingDevice", QVariant::fromValue((void *)device));
+        action->setProperty("renderingDevice",
+                            QVariant::fromValue(static_cast<void *>(const_cast<WbRenderingDevice *>(device))));
```

Comment:
Doesn't look very nice, but is apparently a good practice. See https://stackoverflow.com/questions/51249130/nested-static-cast-and-const-cast